vermouth/data/mappings/modifications.mapping

```diff
@@ -199,3 +199,75 @@ N   BB
 CA  BB
 C   BB
 O   BB
+
+
+
+[ modification ]
+[ from ]
+charmm
+[ to ]
+martini22
+
+[ from blocks ]
+TYRPHOS
+[ to blocks ]
+TYRPHOS
+
+[ from nodes ]
+N
+HN
+CA
+HA
+CB
+HB1
+HB2
+CG
+CD1
+HD1
+CE1
+HE1
+CD2
+HD2
+CE2
+HE2
+C
+O
+[ from edges ]
+CB	CA
+CG	CB
+CD2	CG
+CE1	CD1
+CZ	CE2
+OH	CZ
+N	HN
+N	CA
+C	CA
+CA	HA
+CB	HB1
+CB	HB2
+CD1	HD1
+CD2	HD2
+CE1	HE1
+CE2	HE2
+OH	HH
+O	C
+CD1	CG
+CE1	CZ
+CE2	CD2
+OH P1
+P1 O2
+P1 O3
+P1 O4
+H2 O2
```

Comment:
This doesn't look correct to me. `[ from nodes ]`  and `[ from edges ]` should only list nodes/edges that are not already described by `[ from blocks ]` (i.e. the modification TRYPHOS), and even then only insofar the mapping is different from the unmodified block.

Reply:
BUT modifications need to be connected components and if I start stripping them I get the disconnected component error at some point. If I don't include the anchor to the BB it completly missed the PTM. This one works

Reply:
I'll test it and let you know. Could be I made a typo in one of the earlier toys and this worked because I just copied it all from the rtp file.

Reply:
seems to have worked